demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -221,16 +221,18 @@ export class DemoMeetingApp
   static testVideo: string =
     'https://upload.wikimedia.org/wikipedia/commons/transcoded/c/c0/Big_Buck_Bunny_4K.webm/Big_Buck_Bunny_4K.webm.360p.vp9.webm';
   static readonly LOGGER_BATCH_SIZE: number = 85;
-  static readonly LOGGER_INTERVAL_MS: number = 2000;
+  static readonly LOGGER_INTERVAL_MS: number = 2_000;
```

Comment:
Is this a new JS feature? Can we assume it is well-supported?

Reply:
It's a TypeScript feature, introduced in TS 2.7 in late 2017 or early 2018.